src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -16,6 +17,24 @@
 
 namespace {
 
+std::shared_ptr<ov::Model> cvt_kvcache_to_fp16(const std::shared_ptr<ov::Model>& model) {
```

Comment:
@dmatveev do we still need to stick to the approach w/o `PrePostProcessor` or it can be safely used?

Reply:
The answer - yes